lib/logmsg/logmsg.c

```diff
@@ -862,6 +863,27 @@ log_msg_is_tag_by_name(LogMessage *self, const gchar *name)
 /* structured data elements */
 
 static void
+log_msg_sdata_append_key_escaped(GString *result, const gchar *sstr, gssize len)
+{
+  /* The specification does not have any way to escape keys.
+   * The goal is to create syntactically valid structured data fields. */
+  const guchar *ustr = (const guchar *) sstr;
```

Comment:
Is this necessary? `isascii()` has an `int` parameter, `g_string_append_c` expects a `gchar`.